### ISSUE TITLE
[Translate] [French] To translate or not to translate technical terms

### ISSUE THREAD

**Issue by SamGuay** [alan-turing-institute/the-turing-way#1343]
### Summary
After having read #967, #969, #970, and the [translation guide](https://github.com/alan-turing-institute/the-turing-way/blob/translation/translation_guide.md), I'm starting an issue regarding the difficulty of translating some technical terms from English to French and whether or not we should translate every technical term. 

I opened the [French Transifex](https://www.transifex.com/theturingway/theturingway/translate/#fr) a while ago, but soon realized it wasn't *that* straightforward to translate everything. After having tried to translate the *simple* [Glossary](https://www.transifex.com/theturingway/theturingway/translate/#fr/book-content-po-glossary-pot--translation/287439783), I've come to realize that we might loose some meaning if we translate literally all the technical terms. For example, when talking about Docker, should we stick to Dockerfile or translate it to FichierDocker? Do we stick to packages or use *paquets*? See [this](https://github.com/alan-turing-institute/the-turing-way/issues/969#issuecomment-614295641) for an example about the words _Reproducible_ and _replicable_ in Turkish. 

I guess I just want to make sure that French readers will be able to use the technology with the proper terms, search the web properly to find relevant answers when they have questions, and avoid misinterpretation because of a translation issue. 

Deciding now on most used terms will spare us from unintended consequences downstream. This doesn't mean we can't start now, but I think we should note terms *at risk*.

### What needs to be done?

- [ ] *Make a list of terms we are not sure of the translation (here or elsewhere?)*
    - We can comment and add issues directly on terms and sentences in Transifex, but this means all the discussion would be restricted to the people who work on the translation and not publicly available (like this issue).

### Who can help?

Maybe @tonyyzy & @annahadji ?

I'll also ping the people at @osumontreal as one of our long-term goal is to translate the Turing Way in French, and @jessyprovencher & @Remi-Gau because of #1339.

---

**Comment by Remi-Gau** [comment]
I have to admit that I don't feel too strongly about a certain level of "anglicism" in my French so I am more in favor of sticking with the English version of many of those but I agree that some might require some agreement. 

I guess the extreme cases are easy will be easy to agree on:
- no one will use "Fenêtres" to describe their Windows operating system
- no one will use "mouse" to describe their "souris"

Dockerfile has no transparent equivalent in French that I know of and I fear that using a literal translation will bring confusion when the reader will have to interact with a Dockerfile for the first time.

Package I am less familiar with. The official Python doc seems to use "paquet": so maybe for many terms we can try to see what the "official" doc in French (if it exists) uses?

---

I am tempted to use this issue to discuss this and keep in the top message the list of agreements we have come to (possibly with a link to the post in the issue where this agreement was reached).

**Comment by jessyprovencher** [comment]
Some technical terms could be found at the following link: [http://gdt.oqlf.gouv.qc.ca/index.aspx](http://gdt.oqlf.gouv.qc.ca/index.aspx)

For trademarks terms and extreme cases, I'm on the same page as @Remi-Gau.

**Comment by SamGuay** [comment]
> I have to admit that I don't feel too strongly about a certain level of "anglicism" in my French so I am more in favor of sticking with the English version of many of those but I agree that some might require some agreement.

Good to know, we're all on the same page. We could simply add a disclaimer somewhere in the beginning that goes along the lines of "*We know there are "official" terms in French, but we thought it was beneficial (and less confusing/awkward) to stick to the English terms for some words because these are the terms even French-speaking people use daily. Here's a list of such terms with their French translation, but beware you'll find only the English terms throughout the book.*"  If we can find some terms in http://gdt.oqlf.gouv.qc.ca/index.aspx, that should be easy.

> I am tempted to use this issue to discuss this and keep in the top message the list of agreements we have come to (possibly with a link to the post in the issue where this agreement was reached).

Yes, I was thinking of adding the words below the **## updates** section. 

---

If it were only a matter of personal opinion, I'd use package and not *paquet* mainly because it sounds weird. Maybe we make up a list then vote on the ones on the list? 

--- 

@beapdk, what are your thoughts on this?

**Comment by beapdk** [comment]
I agree that some terms are easier used in english ! As long as we use them  with «» and the rest is in French I don't think it's a problem !
